tests/sqlparser_hive.rs

```diff
@@ -534,6 +534,20 @@ fn parse_use() {
     );
 }
 
+#[test]
+fn test_show() {
+    hive().verified_stmt("SHOW DATABASES");
```

Comment:
It would be better to use `hive_and_generic` since it also works for generic dialects.

Reply:
Fixed